docs/UserGuide.md

```diff
@@ -285,7 +287,6 @@ be found in the jpo-ode/README.md document:
 | Repository     | Visibility | Description             | Source                                            |
 |----------------|------------|-------------------------|---------------------------------------------------|
 | jpo-ode        | public     | Main repository         | <https://github.com/usdot-jpo-ode/jpo-ode>        |
-| jpo-s3-deposit | public     | S3 depositor service    | <https://github.com/usdot-jpo-ode/jpo-s3-deposit> |
```

Comment:
Should we replace this with the utils?

Reply:
Yes, I will update this to include that.